install/Knative-custom-install.md

```diff
@@ -0,0 +1,276 @@
+# Performing a Custom Knative Installation
+
+Use this guide to perform custom installations of Knative on your existing
+Kubernetes clusters. Knative's pluggable components allow you to
+install only what you want in each of your clusters.
+
+The steps covered in the guide are for advanced operators who want to
+customize each Knative installation. Installing individual Knative components
+requires that you to run multiple and separate installation commands.
+
+## Before you begin
+
+* If you are new to Knative, you should instead [install one of the default
+  installation packages](./README.md). Follow the host specific Knative installation
+  instructions for the default package to help you get up and running with the
+  least effort. Installing a default package also ensures that you can run all the
+  [Knative Serving Samples](https://github.com/knative/docs/blob/master/serving/samples/README.md).
+
+* The following steps use `bash` for a MacOS or Linux environment; for Windows,
+  some commands might need adjustment.
+
+* This guide assumes that you have an existing Kubernetes cluster,
+  on which you're comfortable installing and running _alpha_ level software.
+
+* Kubernetes requirements:
+
+  * Your Kubernetes cluster version must be v1.10 or newer.
+
+  * The
+    [MutatingAdmissionWebhook admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-on-an-admission-controller)
```

Comment:
MutatingAdmissionWebhook admission controller can be disabled if end user do not want to use Automatic Sidecar Injection, so here I'd like we update the sentence a bit:

 The
    [MutatingAdmissionWebhook admission controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#how-do-i-turn-on-an-admission-controller) should be enabled on your Kubernetes cluster if you want to use `Automatic Sidecar Injection`.

Reply:
Thanks, after digging into the steps in that k8s link, ive edited these steps further